DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>The docstrings at the beginning could improve. Make sure it's clear what's expected as input, and what will be returned. Use this section of the developer guide as a reference: <a href="https://developer.lsst.io/python/numpydoc.html#documenting-python-apis-with-docstrings" target="_blank">https://developer.lsst.io/python/numpydoc.html#documenting-python-apis-with-docstrings</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Oh yes that needed work. Thanks for pointing me to the developer guide. Hopefully better now.</p>